source/runbooks/rotating-secrets.html.md.erb

```diff
@@ -13,13 +13,31 @@ We don't have many secrets on stored on the Modernisation Platform, but they are
+| GitHub MP CI User PAT | github_ci_user_pat | Used to create PRs etc in GitHub actions and deploy GitHub resources via Terraform | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| GitHub MP CI User Environments Repo PAT | github_ci_user_environments_repo_pat | Used in reusable pipelines of the modernisation-platform-environments repository. This is so that the CI user can post comments in PRs, e.g. tf plan/apply output. | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret.|
+| GitHub MP CI User Password | github_ci_user_password | Used to log in and set the PAT | AWS Secrets Manager | Log in to GitHub as the user and reset the password, update the secret |
+| environment_management | Environment Management | IDs for AWS-specific resources for environment management, such as organizational unit IDs | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| nuke_account_ids | Nuke ID List | Account IDs to be auto-nuked on weekly basis. CAUTION: Any account ID you add here will be automatically nuked! This secret is used by GitHub actions job nuke.yml inside the environments repo, to find the Account IDs to be nuked. |	Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| nuke_account_blocklist | Nuke Block List | Account IDs to be excluded from auto-nuke. AWS-Nuke (https://github.com/rebuy-de/aws-nuke) requires at least one Account ID to be present in this blocklist, while it is recommended to add every production account to this blocklist. | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| pagerduty_userapi_token |	PagerDuty user api | PagerDuty api user token | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| mod-platform-circleci | Circle CI ID | CircleCI organisation ID for ministryofjustice, used for OIDC IAM policies | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| ModernisationPlatformOrganisationManagement IAM user in MoJ root account | N/A | Used to perform limited activities in the root account. No longer used as replaced by OIDC but user kept for breakglass purposes. | Not stored | No active access keys, if keys or password needed contact Operations Engineering |
+| Modernisation Platform Account Root User Password | N/A |Only used during initial platform set up, log in prevented via SCP and no password or keys set | Not stored | Disable or move account to a non SCP protected OU and follow the password reset steps |
+
+## Terraform option
```

Comment:
I don't think this really adds anything and can be removed

Reply:
I'll remove the terraform code if not needed but it does say how to rotate secrets so I thought it may be useful

Reply:
It does, but we don't use this method to rotate any of these secrets

Reply:
Terraform code was removed